roles/nvidia-network-operator/tasks/main.yaml

```diff
@@ -0,0 +1,40 @@
+---
+# Deploy network operator related tasks on master node
+#
+
+- name: label the nodes
+  shell: kubectl label --overwrite nodes {{ item }} node-role.kubernetes.io/worker=
+  with_items: "{{ groups['kube-node']}}"
+
+- name: Install openshift
+  shell: pip3 install openshift
+
+- name: Add helm repo for network operator
+  kubernetes.core.helm_repository:
+    name: mellanox
+    repo_url: "https://mellanox.github.io/network-operator"
+
+- name: Deploy network operator helm chart
+  kubernetes.core.helm:
+    name: network-operator
+    release_namespace: network-operator
+    chart_version: 1.1.0
```

Comment:
These should all be parameterized as variables like we do in the GPU Operator playbook.

Reply:
Good call, will move it to variable file.